pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -63,13 +63,14 @@ type srv struct {
 	upgrader             *websocket.Upgrader
 	tokens               io.DownlinkTokens
 	useTrafficTLSAddress bool
+	wsPingInterval       time.Duration
 }
 
 func (*srv) Protocol() string            { return "basicstation" }
 func (*srv) SupportsDownlinkClaim() bool { return false }
 
 // New creates the Basic Station front end.
-func New(ctx context.Context, server io.Server, useTrafficTLSAddress bool) *echo.Echo {
+func New(ctx context.Context, server io.Server, useTrafficTLSAddress bool, wsPingInterval time.Duration) *echo.Echo {
```

Comment:
If we keep adding config/options here, we may want to consider switching to variadic `Option`s arguments.

Reply:
Then we also need to have defaults if we don't specify those options, while we already have defaults for the component config.

I'd work with an options struct instead to group settings, if the goal is to reduce the size of the argument list.

Reply:
Yup I plan to do this in the future if we add need to add another field.